sdk/storage/Azure.Storage.DataMovement.Files.Shares/src/ShareFileStorageResource.cs

```diff
@@ -17,7 +17,6 @@ internal class ShareFileStorageResource : StorageResourceItemInternal
     {
         internal long? _length;
         internal readonly ShareFileStorageResourceOptions _options;
-        internal ETag? _etagDownloadLock = default;
```

Comment:
Sorry I missed this earlier but was this not being used for Failes currently? Would checking the Etag during multi-part download work the same for Files? If so, maybe we should open an issue to track adding this back? Looks like there is a TODO below indicating it's not possible?

Reply:
Nah you didn't miss it, I missed removing it cause it wasn't even being used anywhere.

I actually looked into using the Etag for multi-part download work, however we can't do the same thing we did for Blobs to check the ETag (e.g. sending the `If-Match` header with the ETag we are expecting).

I will add an issue to look into this future if we can apply this feature in a different way (e.g. maybe we can check the ETag after we downloaded the chunk and confirm that it matches?)